sdk/servicebus/Azure.Messaging.ServiceBus/tests/Receiver/SessionReceiverLiveTests.cs

```diff
@@ -16,24 +16,22 @@ namespace Azure.Messaging.ServiceBus.Tests.Receiver
     public class SessionReceiverLiveTests : ServiceBusLiveTestBase
     {
         [Test]
-        [Ignore("https://github.com/Azure/azure-sdk-for-net/issues/13717")]
-        [TestCase(1, null)]
-        [TestCase(1, "key")]
-        [TestCase(10000, null)]
-        [TestCase(null, null)]
-        public async Task PeekSession(long? sequenceNumber, string partitionKey)
+        [TestCase(1)]
+        [TestCase(10000)]
+        [TestCase(null)]
+        public async Task PeekSession(long? sequenceNumber)
         {
-            await using (var scope = await ServiceBusScope.CreateWithQueue(enablePartitioning: false, enableSession: true))
+            await using (var scope = await ServiceBusScope.CreateWithQueue(enablePartitioning: true, enableSession: true))
```

Comment:
Before, because this test was failing, it was testing this:
> The service made an update to require that sessionId matches partitionKey even on non-partitioned entities.

Since you enabled partitioning now, do we have a test to test this situation for non-partitioned entities?

Reply:
Woops I missed this comment. The test cases we had before didn't really make sense because you shouldn't set the session and partition to different things. As far as non-partitioned tests go, the majority of our tests are on non-partitioned entities.